core/src/main/java/org/bitcoinj/wallet/Wallet.java

```diff
@@ -413,13 +438,36 @@ public static Wallet fromSpendingKey(NetworkParameters params, DeterministicKey
 
     /**
      * Creates a wallet that tracks payments to and from the HD key hierarchy rooted by the given spending key.
-     * The key is specified in base58 notation and the creation time of the key. If you don't know the creation time,
-     * you can pass {@link DeterministicHierarchy#BIP32_STANDARDISATION_TIME_SECS}.
+     * @param params The network
+     * @param spendingKeyB58 The key in base58 notation
+     * @param creationTime creation time of the key (if unknown use {@link #fromWatchingKeyB58(NetworkParameters, String)}
+     * @return a new wallet
+     */
+    public static Wallet fromSpendingKeyB58(NetworkParameters params, String spendingKeyB58, Instant creationTime) {
```

Comment:
Since we've deprecated the old `fromWatchingKeyB58` method, we should probably change the 1st parameter from `NetworkParameters` to `Network` as well.